README.rst

```diff
@@ -73,55 +147,25 @@ Instead of building it yourself, you can also pull an image from Dockerhub
 and then move on to using this image in the next section, where you replace
 ``octimage`` with ``fangohr/octopus:13.0``.
 
+If the ``docker pull`` command is not run, then docker will execute it
+automatically when a ``docker run`` command needs a particular image (such as
+``fangohr/octopus:13.0``).
+
 
 Step 2: Use the Docker image
 ----------------------------
 
 To use the Docker image::
 
   docker run --rm -ti -v $PWD:/io octimage octopus
-  
-Meaning of the switches:
-
-- ``--rm`` remove docker container after it has been carried out (good practice to reduce disk usage).
-- ``-ti`` allow to make the Terminal session Interactive
-- ``-v $PWD:/io``: take the current working directory (``$PWD``) and mount it in
-  the container in the location ``/io``. This is also the default directory of
-  the container.
-- ``octimage`` is the name of the container image. The next 
-- ``octopus`` is the name of the executable to run in the container. 
-
-This is tested and known to work on OSX. On Linux, there is a permissions issue
-if (numerical) user id on the host system and in the container deviate.
-
-If you want to use multiple MPI processes (for example 4), change the above line to::
 
-    docker run --rm -ti -v $PWD:/io octimage mpirun -np 4 octopus
-
-
-Typical workflow with Octopus in container
-------------------------------------------
-
-- edit your ``inp`` file and save it  (on the host computer)
-- call Octopus (in the container) by running ::
-
-      docker run --rm -ti -v $PWD:/io octimage octopus
-  
-  Only the ``octopus`` command will be carried out in the
-  container. Any output files are written to the current directory on the host.
-- carry out data analysis on the host
-
-If you want to work interactively *inside* the container, replace the name of the executable with ``bash``::
-
-  docker run --rm -ti -v $PWD:/io octimage bash
-  
-You are then the root user in the container. Octopus was compiled in ``/opt/octopus*``. There are also some trivial examples in ``/opt/octopus-examples``.
+See Quick start section above for more details.
```

Comment:
Is it possible to somehow put a cross reference in rst? we could then add a hyperlink to that section. I did do a brief search but didn't find a straight forward solution.

Reply:
I also didn't find a solution quickly. The problem is that GitHub does not support all rst features. (There is a solution in general, but it didn't work on GitHub when I tried.)